pkg/dmaasbackup/nonperiodic_backup.go

```diff
@@ -54,7 +38,49 @@ func (d *dmaasBackup) processNonperiodicConfigSchedule(dbkp *v1alpha1.DMaaSBacku
 		bkpName := bkp.Name
 		dbkp.Status.VeleroBackupName = &bkpName
 		d.logger.Infof("Backup=%s created", bkpName)
+		return nil
```

Comment:
since LastBackupStatus has been removed as of now, can we set the status of DMaaSBackup to Completed so that Execute won't get called for this

Reply:
if we do set the status to Completed, we might need to process Finalizer in reconciler (or) better to remove shouldProcessDMaaSBackup which also avoids processing Finalizer which is after that check